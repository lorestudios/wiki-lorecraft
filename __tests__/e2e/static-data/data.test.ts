describe('static data file support in vite 3', () => {
  beforeAll(async () => {
    await goto('/static-data/data')
  })

  test('render correct content', async () => {
    const pLocator = page.locator('.VPContent p')

    const pContents = await pLocator.allTextContents()
    expect(pContents).toMatchSnapshot()
  })
})
