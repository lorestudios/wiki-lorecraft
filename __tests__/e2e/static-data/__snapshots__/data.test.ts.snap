// Vitest Snapshot v1

exports[`static data file support in vite 3 > render correct content 1`] = `
[
  "[
  {
    \\"foo\\": true
  },
  {
    \\"bar\\": true
  }
]",
]
`;
